### DISCUSSION TITLE
애니메이션 효과 추가 건의

### DISCUSSION THREAD

**Discussion by CodeName393** [pronist/hELLO#94] [Ideas]
1. '다크모드->라이트 모드' or '라이트 모드->다크모드'로 전환될때 애니메이션 효과가 있었으면 좋겠습니다.
2. 오른쪽 책갈피 항목 선택후 이동될 때 스크롤 애니메이션 효과가 있었으면 좋겠습니다.
3. 블로그 포스트에서 '하트(좋아요)'를 누르고 하트 모양이 올라오는 애니메이션 효과가 있었으면 좋겠습니다.
4. 이벤트를 줄 수 있는 오브젝트에 커서를 올릴때 나오는 애니메이션 효과를 댓글 작성 버튼등 더욱 많은 곳에 추가했으면 좋겠습니다!

**Comment by jhhan128** [comment]
건의하신 것들 괜찮아 보입니다. 하지만 혹시 추가되게 된다면 사용자가 켜고 끌 수 있게 하는 것이 좋을 것 같습니다.

**Comment by CodeName393** [comment]
> 건의하신 것들 괜찮아 보입니다. 하지만 혹시 추가되게 된다면 사용자가 켜고 끌 수 있게 하는 것이 좋을 것 같습니다.

그것도 좋네요! 원하는 애니메이션 효과만 껏다 켰다 할 수 있으면 더욱 좋을거 같아요!

**Comment by pronist** [comment]
@CodeName393 이슈는 닫아두겠지만, 이 요청은 나중에 추가되거나 혹은 기각될 수 있습니다. :)
